docs/components/field/index.md

```diff
@@ -12,6 +12,10 @@ Field is a component to aid in creating form components. It provides a label, hi
 - `Control`: A control block where a form element is rendered, for example, an `<input>`, `<textarea>`, etc.
 - `Error`: Renders a `<div>` element. Error information is normally rendered here.
 
+## Design Guidelines
+
+- All components should have 0.125rem or 2px of spacing between the elements. This is normally applied by using `mt-0.5` or `space-y-0.5`. It is up to consumers to apply these classes themselves. If using our Toucan `*Field` components, we handle this automatically.
```

Comment:
Suggestion: order of what to change
- `gap` or `space` first
- followed by `margin` as a last resort (ish)